tmp/differences.json

```diff
@@ -0,0 +1,352 @@
+{
+    "common_metrics": {},
```

Comment:
The output is great, but this says there are no metrics in common at all? How then did your previous screenshot-style validation was saying there there were?

Please check it your diff script (in the comments explain how you ran it).

Reply:
The common metrics are query_latency, query_response and query_requests.
The only dfference between them was for V1 the prefix 'jaeger' was attached and for V2 "otelcol" was attached as prefix.

Reply:
I had to personally edit the JSON and add those metric names, because the script won't be able to call that as common.

Reply:
There are not "common", different prefix means a different metric. Why do we prepend otelcol for query metrics in v2? Can we not do that?

Reply:
I didn't have an idea about that. I though that was default behaviour for V1 all metrics have jaeger prepended to them and for V2 all of them have otelcol.
Does it have to do something with the different collector we are using in both scopes?

Reply:
we need to find out. It's possible that OTEL code adds this prefix somewhere.